styles/work-popup.css

```diff
@@ -0,0 +1,223 @@
+
+.modal {
+  background-color: white;
+  position: fixed;
+  top: 5vh;
+  left: 5vw;
+  width: 90vw;
+  height: 90vh;
+  overflow-y: scroll;
+  overflow-x: hidden;
+  border-radius: 10px;
+  filter: blur(0);
+}
+
+.modal-body {
+  display: flex;
+  flex-direction: row;
+}
+
+.modal-content {
```

Comment:
- [ ] There are still issues with responsiveness: 
  - [ ] The text in `See Source` button is overlapping the button when resized the screen from the large screens to small screens